pkg/services/ldap/service/ldap.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sync"
 
 	"github.com/grafana/grafana/pkg/apimachinery/identity"
+	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
```

Comment:
Could you please replace this with `github.com/grafana/grafana/pkg/infra/log`?

Reply:
Good catch, I fixed it.